tests/test_cli_json.py

```
@@ -66,6 +71,73 @@ def _raise_db_locked(*_args: object, **_kwargs: object) -> None:
     raise RuntimeError("db locked")
 
 
+def test_cli_object_helpers_accept_abstract_mappings_and_sequences() -> None:
+    """CLI JSON helpers should keep non-dict mappings and non-list sequences."""
+    object_mapping = cast(
+        Callable[[object], Mapping[str, object]],
+        getattr(cli_module, "_object_mapping"),
```

Ruff (0.15.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)